### PR TITLE
add vpn commands

### PR DESCRIPTION
**Pull request by DavidSpek**
## Summary
Adds commands for listing VPN servers (although there should only be one), list the clients of a particular server, creating a new vpn client for a server and getting the configuration of a client.

## Test Plan
Manually ran all the commands to verify they work as expected. Added some initial tests but more can be added.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/vpn.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+func (p *Plural) handleWireguardPeerCreate(c *cli.Context) error {
+	var serverName string
+	serverName = wireguardServerName
+	if c.String("server") != "" {
+		serverName = c.String("server")
+	}
+
+	name := c.Args().Get(0)
+	conf := config.Read()
+	if err := p.InitKube(); err != nil {
```
Comment (michaeljguarino) [inline cmd/plural/vpn.go:156]: A lot of these functions seem to init kube and checkIfVPNInstalled, perhaps we should make a wrapper func like we have for some other stuff like `rooted` elsewhere to remove duplicate code.

File: cmd/plural/vpn.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+		server = c.String("server")
+	}
+
+	conf := config.Read()
+	if err := p.InitKube(); err != nil {
+		return utils.HighlightError(err)
+	}
+
+	if err := p.checkIfVPNInstalled(); err != nil {
+		return utils.HighlightError(fmt.Errorf("wireguard is not installed. run `plural bundle list wireguard` to find the bundle to install"))
```
Comment (zreigz) [inline cmd/plural/vpn.go:122]: you can extract this sentence to the const, it's used in 4 places

**Review by zreigz — CHANGES_REQUESTED** [review]

**Comment by zreigz** [comment]
Generally, everything works fine :+1:

File: cmd/plural/vpn.go [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+		return utils.HighlightError(err)
+	}
+
+	peerConfig, ok := secret.Data[peer.Status.ConfigRef.Key]
+	if !ok {
+		return utils.HighlightError(fmt.Errorf("peer config not ready yet"))
+	}
+
+	if c.String("path") != "" {
+		err := utils.WriteFile(c.String("path")+peer.Name+".conf", peerConfig)
```
[the remark this answers is not in the document]
Comment (DavidSpek) [reply cmd/plural/vpn.go:237]: Yeah that's probably better. I'll change that.

**Review by zreigz — APPROVED** [review]
lgtm

**Pull request merged**